sonic-net/sonic-buildimage
# [decap] Enable `ipinip.json.j2` template to use new db schema

#### Why I did it
Enable `ipinip.json.j2` to use the new DB schema to create decap tunnel and decap terms.

Signed-off-by: Longxiang Lyu <lolv@microsoft.com>


##### Work item tracking
- Microsoft ADO **(number only)**: 27768450

#### How I did it
Use `TUNNEL_DECAP_TERM_TABLE` to create decap rules.

#### How to verify it
UT and verify on testbed together with https://github.com/sonic-net/sonic-swss/pull/3117.



#### Which release branch to backport (provide reason below if selected)




#### Tested branch (Please provide the tested image version)




#### Description for the changelog




#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
